src/transformers/modeling_utils.py

```diff
@@ -2385,9 +2386,15 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             modules_to_not_convert.extend(keep_in_fp32_modules)
 
             model = replace_8bit_linear(
-                model, threshold=load_in_8bit_threshold, modules_to_not_convert=modules_to_not_convert
+                model,
+                threshold=load_in_8bit_threshold,
+                modules_to_not_convert=modules_to_not_convert,
             )
```

Comment:
Dunno why this is reformatted but should be left as is.

Reply:
It was the extra `,` that caused the reformatting :D